src/Display/Modal/ModalStyle.ts

```diff
@@ -87,7 +87,15 @@ export const ModalContentArea = styled.div<ModalContentAreaProps>`
       `;
     }
   }}
-    
+
+  @media (max-width: 950px) {
```

Comment:
Currently

@media (max-width: ${ScreenSize.mobileM}px) {
    width: 95vw;
  }
This helps for the s and m modal sizes as their width is less than the mobileM and tablet device size.

For the sizes l and xl this criteria should be followed from 950px screen size (as width of xl is 920) as the widths of the two are more than the tablet and mobileL device sizes.

Reply:
Maybe we just need `max-width: 95vw`?

Reply:
Nice suggestion, thanks!